lib/infra/infra-stack.ts

```diff
@@ -740,11 +755,20 @@ export class InfraStack extends Stack {
       });
     }
 
-    const cfnInitConfig: InitElement[] = [
+    if ((nodeType === 'data' || nodeType === 'single-node') && this.useInstanceBasedStorage) {
+      cfnInitConfig.push(InitCommand.shellCommand('set -ex; sudo mkfs -t xfs /dev/nvme1n1; '
+          + 'sudo mkdir /mnt/data; sudo mount /dev/nvme1n1 /mnt/data; sudo chown -R ec2-user:ec2-user /mnt/data',
+      {
+        ignoreErrors: false,
+      }));
+      currentWorkDir = '/mnt/data';
```

Comment:
so does this mean that now opensearch will be installed at `/mnt/data` folder  for both instance based storage? and for all others it will be at /home/ec2-user?

Reply:
yes, if instance based storage then the `nvme1n1` disk will be mounted on `/mnt/data` path and OS will started in that path, if not selected then everything will remain as usual. 
Mounting existing `/home/ec2-user` also works but it is not the ideal way. 
We should be unmouting it first and then mount it again, and by any chance if that disk is busy it will cause failures and make the process cumbersome.

Reply:
Got it make sense. Can we add this information in the readme?